apps/web/app/meetings/_components/Schedule/RoomName.tsx

```diff
@@ -1,15 +1,17 @@
 "use client";
 
+import { type ReactNode } from "react";
+
 interface RoomNameProps {
-  name: string;
+  children: ReactNode;
 }
 
 export default function RoomName(props: RoomNameProps): JSX.Element {
```

Comment:
```suggestion
export default function RoomName({children}:PropsWithChildren): JSX.Element {
```
소소하지만 children만 받는경우에 react의 헬퍼타입을 사용해도 좋았던거같아~

Reply:
헉 그렇네 맞다 나 왜 까먹고있었지...